code/_core/datum/reagent/reagent_nutrition_plants.dm

```diff
@@ -149,3 +149,15 @@
 	return .
 
 
+/reagent/nutrition/lipolicide
+	name = "lipolicide"
+	desc = "Plant-based nutrition-eating germs."
+	color = "#511500"
+
+	nutrition_amount = -5
+	nutrition_quality_amount = -1
```

Comment:
Not sure if this will behave well as a negative number. This should be set to 0 as a precaution because I'm pretty sure it would break things here or best case scenario consider lipocide healthy.